eris.go

```diff
@@ -177,6 +181,12 @@ func (e *rootError) Unwrap() error {
 	return e.ext
 }
 
+// StackFrames returns the trace of a root error in the form of a program counter slice.
+// This method is currently called by external error tracing library (sentry).
```

Comment:
Super minor but I suggest capitalizing Sentry here and below to make it more clear that it's some other library